py-polars/polars/dataframe/frame.py

```diff
@@ -2394,6 +2416,11 @@ def write_ndjson(self, file: IOBase | str | Path | None = None) -> str | None:
         file
             File path to which the result should be written. If set to ``None``
             (default), the output is returned as a string instead.
+        storage_options
+            Important: this only works if file is type string
```

Comment:
Why is this?

Reply:
I am just mentioning it there in case people expect this to work with Pathlib, since Pathlib will remove double slashes in the URI but then the URI path doesn't work anymore.